recipe/meta.yaml

```
@@ -31,16 +31,18 @@ package:
 source:
   - url: https://github.com/opencv/opencv/archive/{{ version }}.tar.gz
     fn: opencv-{{ version }}.tar.gz
-    sha256: 8df0079cdbe179748a18d44731af62a245a45ebf5085223dc03133954c662973
+    sha256: cbf47ecc336d2bff36b0dcd7d6c179a9bb59e805136af6b9670ca944aef889bd
     patches:
       # backport https://github.com/opencv/opencv/pull/21611 (unmerged as of 06/2023)
       - patches_opencv/0001-Add-installation-of-pip-metadata-from-cmake.patch
       - patches_opencv/0002-delete-lines-that-download-opencv.patch
       - patches_opencv/0003-find-pkgconfig-on-windows.patch
       - patches_opencv/0004-fix-detection-for-protobuf-23.x.patch
+      # reverted https://github.com/opencv/opencv/pull/23394 since does not work for macOS 10.9 x86_64
+      - patches_opencv/0005-revert-cocoa-wheel-scroll.patch  # [osx and x86_64]
```

```suggestion
      # - patches_opencv/0005-revert-cocoa-wheel-scroll.patch  # [osx and x86_64]
```

with your increase in the SDK this should be fine right? Thanks for looking into this, I too agree that the default SDK is likely too low, but it has been difficult to change it.

---

yes, PR is ready now.